Alien.h

```diff
@@ -17,3 +20,5 @@ namespace foreign_life {
 
     };
 }
+
+#endif // ALIENBASEBALL_ALIEN_H_
```

Comment:
No newline at the end of the file.
In Settings (Ctrl+,) look for "Files: Insert Final Newline"